homeassistant/components/smhi/weather.py

```diff
@@ -126,6 +126,8 @@ class SmhiWeather(WeatherEntity):
     _attr_native_wind_speed_unit = SPEED_METERS_PER_SECOND
     _attr_native_pressure_unit = PRESSURE_HPA
 
+    _attr_has_entity_name = True
```

Comment:
I'm new to the `field(...)` notation for python so feel free to ignore this comment.

Another PR to update entity naming (#75303) uses `_attr_has_entity_name: bool = field(default=True, init=False)` instead.  Is there any reason this shouldn't do the same thing?

Reply:
The Huaweu LTE integration uses the attrs library. That's not applicable here.